pkg/activator/handler/handler_test.go

```diff
@@ -268,6 +308,159 @@ func TestActivationHandler(t *testing.T) {
 
 }
 
+// Make sure we return http internal server error when the Breaker is overflowed
+func TestActivationHandler_Overflow(t *testing.T) {
+	wantedSuccess := 20
```

Comment:
those are consts?

Reply:
Same below.

Reply:
The var should be fine in this case I think, since we do not reuse the same value. Unless you have a strong reason to make it a const.

Reply:
they don't have to be global consts, ie. 
```
func... {
const (
a = b
c = d
)
...
}
```

Reply:
Sure, if you feel that they would make the code better, I can add them.

Reply:
Constants are constants :) it's more expressive that way.